filesystem/statik.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Pilosa Corp.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filesystem
```

Comment:
same comment as the diagnostics PR - I think this package should be called `statik`

Reply:
the problem here is that we already have a `statik` package that imports `pilosa`, so putting this into that package introduces and import loop.

Reply:
I said that wrong, `pilosa` already imports `statik`.

Reply:
I'll see if I can either rename that other one, or name this one something a little better than `filesystem`.

Reply:
I think that underscore statik import should be moved into the statik(filesystem) subpackage anyway? If it can't be, then most of the point of this exercise is lost.